### PR TITLE
fix #80021 forklog.com

### LINKED ISSUE #80021: forklog.com
### Issue URL (Ads)
 [https://forklog.com/news/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fforklog.com%2Fnews%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Language-specific:</b><br/> AdGuard Russian
Other extensions used: | Kaspersky Internet Security 21.3.10.391(a)

### PR DESCRIPTION
**Pull request by wout-codes**
#80021

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -49,6 +49,8 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+forklog.com##.single_boba
```
Comment (Sergey-Lyapin) [inline RussianFilter/sections/specific.txt:52]: boba is brba now

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -1773,6 +1773,7 @@ revouninstaller.com,logincasino.com,borsagundem.com,officelife.media,kod.ru,mast
 ! ###mc_embed_signup
 kosicednes.sk,tinyprojects.dev,cambio16.com,voxukraine.org,pnas.org,cachevalleydaily.com,bisnis.com,ilpost.it,bsblog.info,alcoparty.com.ua,codastory.com,informazionefiscale.it,kerch.fm,cinemablend.com,prostofinovo.ru,comss.ru,majorgeeks.com,buhgalter.com.ua,providr.com,24tv.ua,babylonbee.com,dexerto.com,football365.com,menrec.com,netdoktor.de,ottawamatters.com,retail-insider.com,walterfootball.com,wweek.com,zozhnik.ru###mc_embed_signup
 !
+forklog.com##.subscribe_blk
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/annoyances.txt:1776]: we need to block `.subscribe_blk_desc` too

`forklog.com##div[class^="subscribe_blk"]`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/annoyances.txt:1776]: I don't know where you have found it. But I will add it anyway.

**Comment by Alex-302** [comment]
@Sergey-Lyapin

File: RussianFilter/sections/specific.txt [hunk 3]
```diff
@@ -49,6 +49,8 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+forklog.com##.single_brba
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt:52]: or `forklog.com##div[style="width: 100%; height: 100%"] > a[target="_blank"] > img`

Comment (wout-codes) [reply RussianFilter/sections/specific.txt:52]: Is `##.single_brba`, enough.

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt]: Screenshot 2021-04-15 at 16 13 09

File: SocialFilter/sections/specific.txt [hunk 4]
```diff
@@ -38,6 +38,8 @@ donbosco-medien.de,giantbomb.com,strefainwestorow.pl,narcity.com,lingohut.com,mt
...
 !
+forklog.com##.share_blk
+forklog.com##.item_share_blk
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:42]: let's unite these rules 
forklog.com##div[class*="share_blk"]

Comment (wout-codes) [reply SocialFilter/sections/specific.txt:42]: @Sergey-Lyapin Why not? `forklog.com##div[class$="share_blk"]`

Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt]: Yes, we can use $= here but new classes are usually added at the end and this rule will not work in this case, so I like *= more

**Pull request merged**